src/linear_algebra/alternating.lean

```diff
@@ -312,6 +312,7 @@ def comp_alternating_map (g : N →ₗ[R] N₂) : alternating_map R M N ι →+
 @[simp] lemma coe_comp_alternating_map (g : N →ₗ[R] N₂) (f : alternating_map R M N ι) :
   ⇑(g.comp_alternating_map f) = g ∘ f := rfl
 
+@[simp]
```

Comment:
I don't understand why this is useful (isn't this lemma true `by simp` via `coe_comp_alternating_map` already?) but if the linter is happy then so am I.